### DISCUSSION TITLE
Question about the slideshow bar

### DISCUSSION THREAD

**Discussion by NitsanCohen770** [prabhuignoto/react-chrono#104] [Q&A]
Is it possible that it will be on the upper (beginning) of the slideshow(vertical mode)?

**Comment by prabhuignoto** [comment]
Unfortunately its not possible at the moment. but this is surely a great idea. i will add this item to the backlog.

If you are a react dev already, feel free to open a PR. let me know if you need help. If not i am happy to do it in the next minor release.
